tools/harness-thci/OpenThread.py

```diff
@@ -1388,7 +1388,6 @@ def ping(self, strDestination, ilength=0, hop_limit=64, timeout=5):
         print('destination: %s' % strDestination)
         cmd = 'ping %s %s 1 1 %d %d' % (strDestination, str(ilength), hop_limit, timeout)
         self.__executeCommand(cmd)
-        time.sleep(1)
```

Comment:
FYI: changing delay after ping could be risky, impacting test results of other test cases.

Reply:
@simonlingoogle do you think increasing the default timeout from 5 to 6 would help there?

Reply:
we used to have sleep to receive ping response. 
But now the ping command should block until the ping response is received. 
So, removing sleep should make sense.